app/helpers/pagination_helper.rb

```diff
@@ -0,0 +1,30 @@
+module PaginationHelper
+  def custom_paginate_renderer(collection, options = {})
+    content_tag(:nav, class: "pagination-controls") do
+      previous_page_link(collection, options) + page_numbers(collection, options) + next_page_link(collection, options)
+    end
+  end
+
+  private
+
+  def link_to_previous_page(collection, options)
+    unless collection.first_page?
+      link_to 'Previous', url_for(page: collection.prev_page), class: options[:previous_class] || "prev-page"
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
      link_to "Previous", url_for(page: collection.prev_page), class: options[:previous_class] || "prev-page"
```